src/lib/viewers/controls/color-picker/__tests__/ColorPickerPalette-test.tsx

```diff
@@ -3,16 +3,31 @@ import { shallow, ShallowWrapper } from 'enzyme';
 import ColorPickerPalette from '../ColorPickerPalette';
 
 describe('ColorPickerPalette', () => {
-    const onColorSelect = jest.fn();
+    const defaultColor = '#fff';
+    const colors = [defaultColor];
 
     const getWrapper = (props = {}): ShallowWrapper =>
-        shallow(<ColorPickerPalette onColorSelect={onColorSelect} {...props} />);
+        shallow(<ColorPickerPalette colors={colors} onSelect={jest.fn()} {...props} />);
 
     describe('render', () => {
-        test('should render the six color swatches', () => {
+        test('should render a single color swatch', () => {
             const wrapper = getWrapper();
 
-            expect(wrapper.find('button').length).toBe(6);
+            expect(wrapper.find('button').length).toBe(1);
+        });
+    });
+
+    describe('onSelect', () => {
+        test('should call onSelect with a button is clicked', () => {
+            const onSelect = jest.fn();
+            const wrapper = getWrapper({ onSelect });
+
+            wrapper
+                .find('button')
```

Comment:
Should we consider adding `testid` instead of using tag names? Should help when we do e2e tests too

Reply:
That sounds good, will add.